docs/src/examples/components/Tooltip/Types/TooltipControlledExample.shorthand.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { Button, Tooltip } from '@stardust-ui/react'
+
+class TooltipControlledExample extends React.Component {
+  state = { open: false }
+
+  handleOpenChange = (e, { open }) => {
+    console.log(`Tooltip requested to change its open state to "${open}".`)
```

Comment:
we usually use alert or we display something on the screen; I don't see this pattern anywhere else

Reply:
Will change this to display something on the screen, the Alert doesn't make sense for the Tooltip, so I changed this to console.log, but it make sense to be visible for the user

Reply:
Reworked this example with knobs.